SQF/dayz_code/actions/plotManagement/maintain_area.sqf

```diff
@@ -11,7 +11,7 @@ s_player_maintain_area_preview = 1;
 
 _target = nearestObject [[player] call FNC_getPos,"Plastic_Pole_EP1_DZ"];
 
-_objectClasses = DZE_maintainClasses;
+_objectClasses = DZE_maintainClasses + DZE_LockableStorage;
```

Comment:
Safes and lockboxes do not need to be maintained. They were not required to be in 1051 and they are not in the default maintain_area.sqf either. Also the SQL event will not set damage on them.